randovania/games/am2r/json_data/Golden Temple.txt

```diff
@@ -650,7 +652,9 @@ Extra - map_name: rm_a1a06
   > Dock to Inner Temple Pipe
       Screw Attack or Disabled Screw Attack Pipe Blocks
   > Door to Parkour Course
-      Can Use Any Bombs
+      All of the following:
+          Can Use Any Bombs
+          After Area 1 - Golden Temple 1 Orb Puzzle or Enabled Entrance Rando
```

Comment:
This is wrong. You can *always* reach the door if you have some bombs. You can only *go* through the door, if you can open the door. And opening the door is put as the dock weakness, which is what gets modified in door weakness rando.
![grafik](https://github.com/randovania/randovania/assets/38186597/8c2637e3-3f83-4a47-b1bc-4ccb49d3203b)

Reply:
Yep I was incorrect, good catch. Will update.